wagtail_bucketav/tests/conftest.py

```diff
@@ -31,3 +33,20 @@ def scan_result_received_receiver():
     scan_result_received.connect(hook)
     yield hook
     scan_result_received.disconnect(hook)
+
+
+@pytest.fixture
+def django_file() -> ContentFile:
+    file = ContentFile(
```

Comment:
Simplified version of what FactoryBoy's Django FileField factory does under the hood :slightly_smiling_face: